src/components/ToastContainer.jsx

```diff
@@ -137,7 +137,7 @@ export class ToastContainer extends React.PureComponent<Props> {
   handleNotify(type, message, title, optionsOverride = {}) {
     if (
       this.props.preventDuplicates &&
-      _.includes(this.state.toastList, { message })
+      this.state.toastList.filter(x => x.message === message).length
```

Comment:
My bad, `_.includes` doesn't support matches shorthand. Would you consider using:
https://lodash.com/docs/4.17.11#findIndex

Reply:
Sure thing, just updated. Should the ToastContainer default change to `false ` as well? Guess it could be problematic for people who've been relying on not preventing duplicates by default.